parity/configuration.rs

```diff
@@ -263,10 +263,11 @@ impl Configuration {
 		self.args.flag_rpcapi.clone().unwrap_or(self.args.flag_jsonrpc_apis.clone())
 	}
 
-	pub fn rpc_cors(&self) -> Option<String> {
-		self.args.flag_jsonrpc_cors.clone().or(self.args.flag_rpccorsdomain.clone())
+	pub fn rpc_cors(&self) -> Vec<String> {
+		let cors = self.args.flag_jsonrpc_cors.clone().or(self.args.flag_rpccorsdomain.clone());
+		cors.map_or_else(Vec::new, |c| c.split(',').map(|s| s.to_owned()).collect())
```

Comment:
What do you think about supporting space-delimited domains as well? It should be another `split` and `flat_map` to make it work.